File: LNPG_01_TIPOS_E_VARIAVEIS/ATIVIDADE_4/C/main.c
#include <stdio.h>

int main() {
    double idade;

    printf("Digite sua idade: ");
    scanf("%lf", &idade);
  
    if (idade >= 18) {
        printf("Você é maior de idade!");
    } else {
      printf("Você é menor de idade!");
    }
  
    return 0;
}

File: LNPG_01_TIPOS_E_VARIAVEIS/ATIVIDADE_4/JAVA/Main.java
package ATIVIDADE_4.JAVA;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        Adulthood checker = new Adulthood();

        System.out.print("Digite sua idade: ");
        int age = scan.nextInt();
        checker.checkAge(age);

        scan.close();
    }
}